front50-sql/src/main/resources/db/changelog/20190415-initial-application-permissions-schema.yml

```diff
@@ -2,6 +2,10 @@ databaseChangeLog:
   - changeSet:
       id: create-application-permissions-table
       author: ajordens
+      preConditions:
+        - onFail: CONTINUE
+        - dbms:
+            type: mysql
```

Comment:
Modifications to an existing changelog will alter its checksum, subsequently causing migrations to fail and thus breaking future deployments of the service - how will existing users of SQL in Front50 upgrade to this version once merged?

Reply:
Are these `preConditions` fields meant to address this?

Do you have any thoughts / guidance for keeping schema changes in-sync across dialects when alterations are needed in the future?

Reply:
> Modifications to an existing changelog will alter its checksum, subsequently causing migrations to fail and thus breaking future deployments of the service - how will existing users of SQL in Front50 upgrade to this version once merged?

Verified that preconditions aren't part of the checksum , so we should be OK.

http://www.liquibase.org/2009/03/what-effects-changeset-checksums.html